addons/account/models/account_payment.py

```diff
@@ -87,6 +87,11 @@ def default_get(self, fields):
         # Check all invoices have the same currency
         if any(inv.currency_id != invoices[0].currency_id for inv in invoices):
             raise UserError(_("In order to pay multiple invoices at once, they must use the same currency."))
+        # Check if, in batch payments, there are not negative invoices and positive invoices
```

Comment:
I would prevent to enter the check if the self.multi == false
(If accessible at this point)